mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py

```diff
@@ -145,7 +145,7 @@ def test_bad_template_does_not_break(self):
                 "name": "Test bad template 1",
                 "model_id": self.env.ref("base.model_res_partner").id,
                 "subtype_ids": [(6, 0, [self.env.ref("mail.mt_comment").id])],
-                "subject_template": "${obaject.number_a} and something",
+                "subject_template": "${object.number_a} and something",
```

Comment:
This is intended. We are trying to render a "bad" template but asserting that the e-mail sending is not blocked. Perhaps we should add a comment explaining that. @victoralmau can you do it?